cps/src/main/resources/org/jenkinsci/plugins/workflow/cps/CpsFlowDefinition/config.jelly

```diff
@@ -25,10 +25,14 @@
 
 <j:jelly xmlns:j="jelly:core" xmlns:f="/lib/form" xmlns:st="jelly:stapler">
   <f:entry title="${%Script}" field="script">
-    <f:textarea checkMethod="post"/>
+    <div id="workflow-editor-wrapper" style="display: none; position: relative">
```

Comment:
That `display` property gets removed via JavaScript once the component is initialised.